src/roles/actors.rs

```diff
@@ -277,25 +287,23 @@ decl_module! {
                     for actor in accounts.into_iter().map(|account| Self::actor_by_account_id(account)) {
                         if let Some(actor) = actor {
                             if now > actor.joined_at + params.reward_period {
-                                // send reward to member account - not the actor account
-                                if let Ok(member_account) = T::Members::lookup_account_by_member_id(actor.member_id) {
-                                    let _ = T::Currency::deposit_into_existing(&member_account, params.reward);
+                                // reward can top up balance if it is below minimum stake requirement
```

Comment:
This looks like the right change, but for two comments:

1. The reward must still exceed the balance used up during the lifetime of a block for anything to ever be paid out to the member *and* to guarantee that the actor can continue acting. I'm not sure this is enough, therefore.
1. The effect of this is that busy actors actually produce less reward for their member than idle actors. I think that's the opposite of what's intended.

Maybe it's enough for getting Athens up and running, because the storage provider incentives are really only properly to be added in future. But it really doesn't seem entirely complete at this point.

Reply:
As we discussed IRL, these edge cases are correct, but would have to be solved by the working group in selecting the best set of role parameters to ensure storage providers are earning more than than their tx fees, as well as introducing a reward model that takes into account how busy the actors are.

Reply:
My main concern for Athens is really that storage providers don't run out of balance for transaction fees - I guess it's okay for now to require people to pay attention.